python/src/FisherSnedecorFactory_doc.i.in

```diff
@@ -61,7 +61,7 @@ are estimated from a sample.
 >>> size = 10000
 >>> distribution = ot.FisherSnedecor(4.5, 8.4)
 >>> sample = distribution.getSample(size)
->>> factory = ot.TruncatedNormalFactory()
+>>> factory = ot.FisherSnedecorFactory()
 >>> estimated = factory.build(sample)
```

Comment:
This change triggers an error in my barnch https://github.com/openturns/openturns/pull/2068